packages/documentation-site/patternfly-docs/content/design-guidelines/styles/theming/theming.md

```diff
@@ -68,6 +72,24 @@ There are a couple of instances when you might want to adjust an existing Patter
 
 Our Figma libraries fully support theming. Designers can create a single layout and then use various Figma "Appearance" settings to swap a wireframe to light, dark, or high contrast mode. This makes it easy to visually test and validate designs across all supported themes.
```

Comment:
Technically, you switch between the modes using the "Apply Variable Mode" toggles in the Appearance section of the component properties panel. but not sure if we really need to be that specific here -- assume our designers probably know this already?

Reply:
I was debating how specific to be, but I like the way you said it, so I'll update this!